### PR TITLE
Adding the basic repo architecture

### PR DESCRIPTION
**Pull request by pechouc**
The aim of this PR is to add a basic architecture to the repository. It is a mix of what was already in the repo (LICENSE, README.md and .gitignore) and of a Python package template, slightly adapted following [this Slack thread](https://data-for-good.slack.com/archives/C01BB6H16FM/p1601497845010600).

### CODE REVIEW

**Review by VincGargasson — DISMISSED** [review]
Looks like to be a good base to start to work on. :)

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pip>=9
+setuptools>=26
+wheel>=0.29
```
Comment (frgfm) [inline requirements.txt:3]: Are those really required? They are always expected to be up to date but comes as base python setup

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+pytest
+pandas
+numpy
+dash
+dash-core-components 
+dash-html-components
+plotly
```
Comment (frgfm) [inline requirements.txt:10]: Better version these requirements (at least a lower bound) ;)

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	@rm -fr pyronear-webapp-*.dist-info
+	@rm -fr pyronear-webapp.egg-info
+
+install:
+	@pip install . -U
+
+all: clean install test black check_code
+
+
+uninstal:
```
Comment (frgfm) [inline Makefile:34]: small typo over there

**Review by frgfm — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks good to me, apart from `.DS_Store` files and the requirements (lower bound versioning at least). Would you mind removing them please? (better add them into the `.gitignore` to avoid this later on)

File: requirements.txt (hunk 1 above)
Comment (pechouc) [reply requirements.txt:3]: `pip` and `setuptools` do but I am not sure for `wheel`.

Typically, the logs of CI checks show that `wheel` was downloaded and installed based on the requirements (requirement was not already satisfied). 

I am removing the two first ones, what do you think about `wheel`? Would you still remove it?

File: Makefile (hunk 3 above)
Comment (pechouc) [reply Makefile:34]: Nice catch, thanks! It is corrected.

File: requirements.txt (hunk 1 above)
Comment (frgfm) [reply requirements.txt:3]: No need to bother, we can check later on, I was mostly curious :sweat_smile:

File: requirements.txt (hunk 2 above)
Comment (pechouc) [reply requirements.txt:10]: I added the versions I usually use as a lower bound

**Comment by pechouc** [comment]
I have pushed a new commit that reflects your comment and re-requested your review but please let me know if that's not the process to follow 😅

**Review by frgfm — CHANGES_REQUESTED** [review]
Thanks for the quick edits! One last thing: you missed the `.github/.DS_Store` file ;)

**Review by frgfm — APPROVED** [review]
Thanks a lot for the PR and the edits!

**Comment by frgfm** [comment]
Oops I updated the repo settings! For next PRs, it's better to use "Squash & merge" rather than basic Merge. The reason behind this is that it enforces each commit on master to have passed the tests :+1:

**Comment by pechouc** [comment]
Very clear! Sorry about that, I'll keep that in mind for next time 😊

**Pull request merged**